tests/protractor/e2e/api/controllers/export-data-2.js

```diff
@@ -141,7 +141,7 @@ describe('Export Data V2.0', () => {
           //
           const expected = [
             '_id,form,patient_id,reported_date,from,contact.name,contact.parent.name,contact.parent.parent.name,contact.parent.parent.parent.name,wd_array,wd_emptyString,wd_false,wd_naughtyArray,wd_naughtyString,wd_null,wd_zero',
-            '"export-data-2-test-doc-4","weird-data-types",,,,,,,,"[0,1,2]","",false,"[0,{\\"foo\\":false,\\"bar\\":null},\\"Hello, \\\\"world\\\\"\\"]","Woah there, \\"Jimmy O\'Tool\\"",,0',
+            '"export-data-2-test-doc-4","weird-data-types",,"",,,,,,"[0,1,2]","",false,"[0,{\\"foo\\":false,\\"bar\\":null},\\"Hello, \\\\"world\\\\"\\"]","Woah there, \\"Jimmy O\'Tool\\"",,0',
```

Comment:
Did you work out why this changed?

Reply:
The dateFormat function returns `''` when date is `null`. Because that's a string the escape code wraps it in quotes.